pkg/nfd-master/nfd-master.go

```diff
@@ -792,16 +838,14 @@ func (m *nfdMaster) nfdAPIUpdateOneNode(cli k8sclient.Interface, node *corev1.No
 		return objs[i].Namespace < objs[j].Namespace
 	})
 
-	klog.V(1).InfoS("processing of node initiated by NodeFeature API", "nodeName", node.Name)
-
-	features := nfdv1alpha1.NewNodeFeatureSpec()
+	klog.V(1).InfoS("processing of node initiated by", "nodeName", nodeName)
```

Comment:
Hmm, this log message looks pretty useless to me (and this new version looks confusing). Prolly dates back to some experiments with the NodeFeatureAPI. I think we could just ditch this msg in this PR

```suggestion
```